### DISCUSSION TITLE
Add a new function like  validate_password

### DISCUSSION THREAD

**Discussion by dengivseti** [fastapi-users/fastapi-users#749] [Ideas]
Hi, please add a new function similar to the validate_password. In addition to the password, we can also validate mail, add a list of banned usernames. She can be called simply validator(user: Union[UserCreate, UserDB])

**Comment by frankie567** [comment]
That's not something I'm currently planning but I think this is something fairly easy to do by overloading some methods in your `UserManager`. For example:

```py
class UserManager(BaseUserManager[UserCreate, UserDB]):
    user_db_model = UserDB
    reset_password_token_secret = SECRET
    verification_token_secret = SECRET

    async def create(
        self, user: UserDB, safe: bool = False, request: Optional[Request] = None
    ) -> UserDB:
        if user.email in ["banned@email.com", "hacker@email.com"]:
            raise HTTPException(status_code=400)
        return super().create(user, safe, request)
```
